<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Footprint Options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Footprint</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Enter the name of the footprint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtFootprintName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Or choose one from this list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbFootprints"/>
       </item>
       <item>
        <widget class="QCheckBox" name="cbxDoWork">
         <property name="text">
          <string>Carry on with footprint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>(Untick this if you simply want to view previous results)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mappings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Tasks</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="cmbCategories"/>
             </item>
             <item>
              <widget class="QTableWidget" name="tblTasks">
               <column>
                <property name="text">
                 <string>Category</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Task Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtTaskName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="txtTaskDescription"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>File Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtFileName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Uses Metasploit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbxUsesMetasploit">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Is Recursive</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbxIsRecursive">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbxTaskEnabled">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="btnCreateTask">
               <property name="text">
                <string>Create Task</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSaveTasksChanges">
               <property name="text">
                <string>Save Changes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Trigger Descriptions</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QTableWidget" name="tblTriggers">
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Description</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string># of events</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="minimumSize">
                    <size>
                     <width>110</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>110</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txtTriggerName"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="minimumSize">
                    <size>
                     <width>110</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>110</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Description</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="txtTriggerDescription"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Events</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QComboBox" name="cmbTriggers"/>
             </item>
             <item>
              <widget class="QTableWidget" name="tblEvents">
               <column>
                <property name="text">
                 <string>Trigger</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value Mask</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Task Category</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Task Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Enabled</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Value Mask</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtEventValueMask"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbxEventEnabled">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QPushButton" name="btnAddEvent">
               <property name="text">
                <string>Add Event</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSaveEvent">
               <property name="text">
                <string>Save Changes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Scope</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Known Hosts</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtKnownHosts"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>Known Ranges</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtKnownRanges"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Exclude List</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtExclude"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
